src/components/TopNav/NavMenu.js

```diff
@@ -8,26 +8,20 @@ import NavigationMenu from 'material-ui/svg-icons/navigation/menu'
 import IconButton from 'material-ui/IconButton'
 import * as auth from '../../services/authNService'
 
-export const mapStateToProps = (state) => {
-    return {
-        alias: state.auth.userAlias
-    }
-}
 
-export const NavMenu = ({dispatch, alias}) =>
+
+export const NavMenu = dispatch =>
```

Comment:
Changing from a [destructuring function signature](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) to a plain args signature is almost always a breaking change; this version of NavMenu will receive as an argument an object named 'dispatch' with a property 'dispatch.dispatch' that's a function. That breaks the LogOut link.

`export const NavMenu = ({dispatch}) =>`

will fix that issue.